samples/audio/sof/sample.yaml

```diff
@@ -0,0 +1,11 @@
+sample:
+  description: Audio with SOF
+  name: sof
+common:
+    tags: audio
+tests:
+  sample.audio.sof:
+    tags: sof
+    build_only: true
+    integration_platforms:
+      - intel_adsp_cavs15
```

Comment:
@lyakh fyi - can you add cavs18 - 25 here once you are happy all is working.

Reply:
@lgirdwood is this for CI? And it seems to be only testing building, so, yes, should be good to add others too. @nashif in an additional PR or in this one?